app/views/groups/edit_invitations.html.erb

```diff
@@ -1,3 +1,6 @@
+<%= provide(:title, "Edit invitations - " + @group.name) %>
```

Comment:
Same as with draws, let's use `"#{@group.name} - Edit invitations"` for this and similar for the other group views.